docs/pages/repo/docs/support.mdx

```diff
@@ -0,0 +1,66 @@
+---
+description: Our Support Policy for Turborepo and its ecosystem
+headeronly: true
+container: true
+searchable: false
+---
+
+## Support Policy
+
+`turbo` is generally supported on Ubuntu, macOS, and Windows on both x86_64 and ARM 64 architectures.
+Specificaly, we build and ship the following binaries via `npm`:
+
+- turbo-darwin-64
+- turbo-darwin-arm64
+- turbo-linux-64
+- turbo-linux-arm64
+- turbo-windows-64
+- turbo-windows-arm64
+
+The darwin builds are intended for macOS on Intel and Apple Silicon. The linux
+builds are intended to run on Ubuntu, but we do not exercise them on other
```

Comment:
Ehh not quite true. We do make sure it works for Alpine because that's what we use at Vercel

Reply:
yeah i wanted to add some more details here and could use help:

- [`turbo/bin` uses `process.platform`][1] to determine which native binary to download
- [`process.platform`][2] can be 'aix', 'darwin', 'freebsd', 'linux', 'openbsd', 'sunos', 'win32`. but we only build a binary for darwin/linux/win32 ones.

I'm guessing `process.platform` is `"linux"` on Alpine, but are there other Linux distros that we intend to capture with this? 

Also, we only build `turbo-linux-64` and `turbo-linux-arm64`, are there any other [`process.arch`][3] that we should support?

[1]: https://github.com/vercel/turbo/blob/51bba18db76ac3f0b60bab9dfc823a6e066642b8/packages/turbo/bin/turbo#L69-L81
[2]: https://nodejs.org/api/process.html#processplatform
[3]: https://nodejs.org/api/process.html#processarch

Reply:
Hmm I'm not exactly sure which Linux distros we support (any idea @gsoltis?). I suspect definitively Alpine and Debian/Ubuntu but probably not much more.

As for arch, I think x86-64 and aarch64 are good enough. Realistically once we move to full Rust, someone could build turbo from source with little effort (although `ring` will give them trouble because it uses platform specific code or even inline assembly)

Reply:
I just ran into someone on twitter saying Alpine doesn't work out of box https://twitter.com/perdanafamul/status/1717507938229837871

English translation from Google:
> But this turborepo doesn't support Alpine, bro, I read the discussions in the issue thread, so what package do you need to install first? I forgot.

Reply:
updated this to soemthing more general for now